ravenframework/Models/EnsembleModel.py

```diff
@@ -136,7 +136,7 @@ def localInputAndChecks(self,xmlNode):
           self.raiseAnError(IOError, "Input XML node for Model" + modelName +" has not been inputted!")
         if len(self.modelsInputDictionary[modelName].values()) > allowedEntriesLen:
           self.raiseAnError(IOError, "TargetEvaluation, Input and metadataToTransfer XML blocks are the only XML sub-blocks allowed!")
-        if child.attrib['type'].strip() == "Code":
+        if child.attrib['type'].strip() in ["Code", 'HybridModel', 'LogicalModel']:
```

Comment:
very ugly :(

Reply:
It would be slightly less ugly if you used a set:
```
       if child.attrib['type'].strip() in {"Code", 'HybridModel', 'LogicalModel'}:
```

Reply:
done